backend/api/controllers/userController.js

```diff
@@ -90,6 +90,34 @@ exports.sign_in = async function (req, res) {
 			});
 		});
 };
+exports.forgot =async function (req, res, next) {
```

Comment:
change the function name to forgotPassword.

Reply:
Please reject this PR I will send it again.

Reply:
done

Reply:
Please cheak it and if you find any issue revert back to me
Thank You!